src/drivers/uavcan/dsdl/com/hex/equipment/flow/20200.Measurement.uavcan

```diff
@@ -0,0 +1,4 @@
+float32 integration_interval    # Integration Interval in seconds
+float32[2] rate_gyro_integral   # Integrated Gyro Data in radians
```

Comment:
Add better comments about axis

Reply:
We should probably keep it identical in case it ever needs to update.
https://github.com/OpenMotorDrive/flow/blob/master/dsdl/com/hex/equipment/flow/20200.Measurement.uavcan.

Reply:
Replaced with the source message. Should we comment it properly upstream? https://github.com/OpenMotorDrive/flow/blob/master/dsdl/com/hex/equipment/flow/20200.Measurement.uavcan